yarn.lock

```diff
@@ -1,11741 +0,0 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Let's bring yarn.lock back. Both npm and yarn locks should be present in the project.